build/docker/alpine.Dockerfile

```diff
@@ -19,23 +19,23 @@ ENTRYPOINT [ "debricked",  "scan" ]
 
 FROM cli AS resolution
 RUN apk --no-cache --update add \
-    openjdk8-jre \
+    openjdk11-jre \
```

Comment:
@viktigpetterr any reason to why we used such an ancient version of Java? Minimum version in Debian is 11 now

Reply:
No there is not, let us go with 11 instead